stride/plotting/plot_traces.py

```diff
@@ -149,7 +258,7 @@ def plot_gather(*args, skip=1, time_range=None, norm=True, norm_trace=True,
     trace_axis = [str(each) for each in trace_axis]
 
     axis.set_xticks(shift.flatten()[::2])
-    axis.set_xticklabels(trace_axis[::2])
+    axis.set_xticklabels(trace_axis[::2], rotation='vertical')
```

Comment:
What does this do?

Reply:
it turns the labels around so they are oriented vertically, it improves readability